react/Badge/Badge.js

```diff
@@ -4,13 +4,14 @@ import React, { PureComponent } from 'react';
 import classnames from 'classnames';
 
 import styles from './Badge.less';
+import { TONE } from '../private/tone';
 
 type Tone =
   | 'accent'
-  | 'critical'
-  | 'info'
-  | 'neutral'
-  | 'positive'
+  | typeof TONE.CRITICAL
+  | typeof TONE.INFO
+  | typeof TONE.NEUTRAL
+  | typeof TONE.POSITIVE
   | 'secondary';
```

Comment:
Are accent and secondary only specific for this component? Should we move them to tones?